ui/main/src/assets/i18n/en.json

```diff
@@ -387,7 +387,6 @@
         "editTitle": "EDIT A NEW CARD",
         "error": {
             "notAvailable": "You are not allowed to send card.",
```

Comment:
Use the opportunity to change this translation to   "notAvailable": "You are not allowed to send card**s**."

It also needs to be changed in tests